eng/restore/netcoreapp/Configurations.props

```diff
@@ -3,6 +3,7 @@
     <BuildConfigurations>
       netcoreapp2.0;
```

Comment:
At what point can we drop `netcoreapp2.0` here? That has been EOL for a while now.

Reply:
EOL doesn't mean that we are dropping nuget assets AFAIK. What we can do here is to harvest that asset instead.

Reply:
Right, we don't want to drop the asset entirely, but we should probably harvest instead of live building.